packages/@okta/vuepress-site/docs/guides/migrate-to-okta-prerequisites/main/index.md

```diff
@@ -1,6 +1,21 @@
 ---
-title: Prerequisites
+title: Migrate to Okta — Prerequisites
+meta:
+  - name: description
+    content: Create a plan for migrating existing users to Okta.
+layout: Guides
 ---
+
+This guide provides you with information to create a plan for migrating existing users to Okta.
+
+---
+
+**Learning outcomes**
+
+Create a plan for migrating existing users to Okta.
+
+---
+
 ## How to plan your migration
```

Comment:
I was out sick on Friday, but I think the team decided against having a header for sections immediately after the nut facts.
```suggestion
```